makepkg.py

```diff
@@ -25,51 +25,81 @@
     }}
 }}]'''
 
+PUBLISH_MANIFEST = '''<?xml version="1.0" encoding="utf-8" ?>
+<ExtensionImage xmlns="http://schemas.microsoft.com/windowsazure"  xmlns:i="http://www.w3.org/2001/XMLSchema-instance">
+  <!-- WARNING: Ordering of fields matter in this file. -->
+  <ProviderNameSpace>Microsoft.OSTCLinuxAgent</ProviderNameSpace>
+  <Type>Test</Type>
+  <Version>{0}</Version>
+  <Label>Microsoft Azure Guest Agent for Linux IaaS</Label>
+  <HostingResources>VmRole</HostingResources>
+  <MediaLink></MediaLink>
+  <Description>Microsoft Azure Guest Agent for Linux IaaS</Description>
+  <IsInternalExtension>true</IsInternalExtension>
+  <Eula>https://github.com/Azure/WALinuxAgent/blob/2.1/LICENSE.txt</Eula>
+  <PrivacyUri>https://github.com/Azure/WALinuxAgent/blob/2.1/LICENSE.txt</PrivacyUri>
+  <HomepageUri>https://github.com/Azure/WALinuxAgent</HomepageUri>
+  <IsJsonExtension>true</IsJsonExtension>
+  <CompanyName>Microsoft</CompanyName>
+  <SupportedOS>Linux</SupportedOS>
+  <Regions>South Central US</Regions>
```

Comment:
maybe parametrize this as well so that global rollout can be done? I'm not sure how they do global rollout for a version when I tried dropping this element or leaving it empty, I got errors.